### PR TITLE
Removed unnecessary int cast in _check_termination of ntheory/factor_.py #26762

### PR DESCRIPTION
**Pull request by jamestitus299**
Removed unnecessary int cast in _check_termination of ntheory/factor_.py #26762

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by arnabnandikgp** [comment]
Hey, @jamestitus299 follow https://docs.sympy.org/dev/contributing/new-contributors-guide/workflow-process.html#mailmap-instructions to add your name as an author in mailmap.

**Comment by haru-44** [comment]
It may be a repetition of what has already been pointed out, but please add the username and email address at the time of commit to the `.mailmap` and run `bin/mailmap_check.py`. If you have any questions, please feel free to ask.

**Comment by jamestitus299** [comment]
bin/mailmap_check.py  tells me to add two jamestitus299 <jamestitus299@gmail.com> and 
jamestitus299 <titusjames299@gmail.com>  into the .mailmap

**Comment by haru-44** [comment]
https://github.com/sympy/sympy/blob/84b114cbed65788e62f46cdd468456f2f876b74f/.mailmap#L142-L148

File: .mailmap [hunk 1]
```diff
@@ -1593,6 +1593,7 @@ gregmedlock <gmedlo@gmail.com>
 helo9 <helo9@users.noreply.github.com>
 hm <hacman0@gmail.com>
 immerrr <immerrr@gmail.com>
+jamestitus299 <titusjames299@gmail.com> jamestitus299 <jamestitus299@gmail.com>
```
Comment (oscarbenjamin) [inline .mailmap:1596]: Take a look at the [AUTHORS](https://github.com/sympy/sympy/blob/master/AUTHORS) file.

Do you want your name to be recorded there as "jamestitus299" or as something like "James Titus"?

**Comment by oscarbenjamin** [comment]
Looks good. Thanks!

**Pull request merged**